tests/Kernel/fixtures/rendering.yml

```diff
@@ -1456,3 +1456,59 @@
       'figcaption.ecl-media-container__caption': 0
     equals:
       'h2.ecl-u-type-heading-2': "Heading"
+- array:
+    '#type': pattern
+    '#id': icons_with_text
+    '#fields':
+      items:
+        -
+          icon: "file"
+          text: "Culture | Innovation"
+        -
+          icon: "calendar"
+          text: "15-16 November 2019, 08:00 AM"
+        -
+          icon: "location"
+          text: "Brussels, Belgium"
+        -
+          icon: "livestreaming"
+          text: "Live streaming available"
+  assertions:
+    count:
+      'ul.ecl-unordered-list--no-bullet': 1
+      'ul li.ecl-unordered-list__item': 4
+      'ul li.ecl-unordered-list__item svg': 4
+    equals:
+      'ul li.ecl-unordered-list__item:nth-child(1)': "Culture | Innovation"
+      'ul li.ecl-unordered-list__item:nth-child(2)': "15-16 November 2019, 08:00 AM"
+      'ul li.ecl-unordered-list__item:nth-child(3)': "Brussels, Belgium"
+      'ul li.ecl-unordered-list__item:nth-child(4)': "Live streaming available"
+- array:
+    '#type': pattern
+    '#id': icons_with_text
+  assertions:
+    count:
+      'ul.ecl-unordered-list--no-bullet': 0
+- array:
+    '#type': pattern
+    '#id': icons_with_text
+    '#fields':
+      items:
+        -
+          icon: "file"
+          text: "Culture | Innovation"
+        -
+          text: "15-16 November 2019, 08:00 AM"
+        -
+          icon: "location"
+        -
+          icon: "livestreaming"
+          text: "Live streaming available"
+  assertions:
+    count:
+      'ul.ecl-unordered-list--no-bullet': 1
+      'ul li.ecl-unordered-list__item': 2
+      'ul li.ecl-unordered-list__item svg': 2
+    equals:
+      'ul li.ecl-unordered-list__item:nth-child(1)': "Culture | Innovation"
+      'ul li.ecl-unordered-list__item:nth-child(2)': "Live streaming available"
```

Comment:
Assert also that we have a `document` icon if none is specified, in:
```
        -
          text: "15-16 November 2019, 08:00 AM"
```

Reply:
Done. but with `file` icon.